link.go

```diff
@@ -50,7 +50,9 @@ type LinkAttrs struct {
 	GROMaxSize   uint32
 	Vfs          []VfInfo // virtual functions available on link
 	Group        uint32
-	Slave        LinkSlave
+
```

Comment:
I assume you added this empty line to avoid the reformat of all existing fields?
Would name the new field `PermHWAddr` good enough?